### DISCUSSION TITLE
Need help figuring out manual sorting

### DISCUSSION THREAD

**Discussion by halindraprakoso** [prisma/prisma#14139] [Q&A] [answered]
Hello,
I need help figuring out an effective way to do manual sorting. Say for example I have a list of fruits and the order is defined by the user based on their favorites via clicking either up/down arrow or even drag-and-drop, how do I make the mutation?

**Comment by halindraprakoso** [comment, marked as answer]
well it's actually not that complicated...

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
